client/flutter/lib/pages/protect_yourself.dart

```diff
@@ -55,10 +57,26 @@ class _ProtectYourselfState extends State<ProtectYourself> {
   }
 
   List<Widget> _buildCards() {
+    final TextStyle normalText = TextStyle(
+      color: Constants.textColor,
+      fontSize: 16 * MediaQuery.textScaleFactorOf(context),
+      height: 1.4,
+    );
+    final TextStyle boldText = normalText.copyWith(fontWeight: FontWeight.w700);
     return (_factContent?.items ?? []).map((fact) {
-      print("fact body = ${fact.body}, image = ${fact.imageName}");
       return _ProtectYourselfCard(
-        message: _message(fact.body),
+        message: Html(
+            data: fact.body ?? "",
```

Comment:
since this variable is used as html, rename to fact.bodyHtml

Reply:
We have 7 or 8 other fields in those `Item` classes that all contain HTML.  Do you want to  adopt that naming convention for them all now? (I'm not sure that we are consistent with this convention throughout the rest of the code).